src/api/app/views/webui2/webui/request/_show_overview.html.haml

```diff
@@ -0,0 +1,12 @@
+%h3
+  %p #{pagetitle} (#{state})
+
+#description-text
+  - if description.blank?
+    %i No description set
+  - else
+    = description_wrapper(toggle_sliced_text(description))
+- if can_add_reviews
+  %p
+    %i.fas.fa-plus-circle.text-primary
+      = link_to('Add a review', add_reviewer_dialog_path(number: number), data: { toggle: 'modal', target: '#add-reviewer-modal' })
```

Comment:
The font of the link text looks different to what we usually use. I think this changes if you swap around icon and link, like this:

```ruby
= link_to(add_reviewer_dialog_path(number: number), data: { toggle: 'modal', target: '#add-reviewer-modal' }) do
  %i.fas.fa-plus-circle.text-primary
    Add a review
```

![2019-02-05_14-34](https://user-images.githubusercontent.com/968949/52276765-5107c980-2953-11e9-949b-7b8b972763c0.png)